pkgs/servers/search/elasticsearch/default.nix

```diff
@@ -19,14 +19,15 @@ stdenv.mkDerivation (rec {
     url = "https://artifacts.elastic.co/downloads/elasticsearch/${name}.tar.gz";
     sha256 =
       if enableUnfree
-      then "0960ak602pm95p2mha9cb1mrwdky8pfw3y89r2v4zpr5n730hmnh"
-      else "1i4i1ai75bf8k0zd1qf8x0bavrm8rcw13xdim443zza09w95ypk4";
+      then "096i8xiy7mfwlslym9mkjb2f5vqdcqhk65583526rcybqxc2zkqp"
+      else "0j3q02c4rw8272w07hm64sk5ssmj4gj8s3qigsbrq5pgf8b03fvs";
   };
 
   patches = [ ./es-home-6.x.patch ];
 
   postPatch = ''
     sed -i "s|ES_CLASSPATH=\"\$ES_HOME/lib/\*\"|ES_CLASSPATH=\"$out/lib/*\"|" ./bin/elasticsearch-env
+    sed -i "s|ES_CLASSPATH=\"\$ES_CLASSPATH:\$ES_HOME/\$additional_classpath_directory/\*\"|ES_CLASSPATH=\"\$ES_CLASSPATH:$out/\$additional_classpath_directory/\*\"|" ./bin/elasticsearch-cli
```

Comment:
How about using `substituteInPlace` for both? I don't spot regexes here and it should be more readable.

Reply:
I've implemented your suggestion and it's more readable indeed.